### ISSUE TITLE
Holy nova - threat (aggro) (BB #492)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#492]
This issue was migrated from bitbucket.
**Original Reporter:** D_Jakob
**Original Date:** 04.02.2014 20:26:46 GMT+0000
**Original Priority:** minor
**Original Type:** bug
**Original State:** closed
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/492

<hr>

This spell creates threat/aggro, In spell description u can read "all effects don't create aggro/threat".

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 03.03.2014 15:35:36 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/492/#comment-8884505

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 16.10.2014 15:56:00 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/492/#comment-13027485

<hr>

Could someone try the below code and tell me if it solves the issue?

```
#!sql

INSERT INTO spell_threat VALUES
(15237, 0), -- Holy Nova effHeal
(23455, 0); -- Holy Nova effDamage
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 17.10.2014 04:08:20 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/492/#comment-13036754

<hr>

Okay, the heal threat works but not the damage. I think that's because we calculate dmg -> threat. We should have the core handle special cases from the spell_threat table.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.10.2014 14:48:38 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/492/#comment-13320733

<hr>

[Core/Threat] Update spell threat calculations.
              Partially implement SPELL_ATTR_EX_NO_THREAT and SPELL_ATTR_EX3_NO_INITIAL_AGGRO.
              Energizes now generate threat.
              Threat distributed amongst enemies / allies.
              Restructure spell_threat table.

```
          By all means, Threat isn't perfect now. The database values need
          to be tweaked heavily to achieve perfect threat calculation.

          Closes #492

          Based on Sarjuuks work.
```

→ <<cset 135b4930594f>>
